src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,146 @@
+import { useContext, useEffect } from 'react';
+import { SearchLink } from './SearchLink';
+import { PeopleContext } from '../contsxts/PeopleContext';
```

Comment:
The import path for `PeopleContext` is incorrect. It should be '../contexts/PeopleContext' instead of '../contsxts/PeopleContext'.